models/review.js

```diff
@@ -13,14 +13,22 @@ const reviewSchema = new Schema(
       min: 1,
       max: 5
     },
-    tutor: {
+    authorId: {
       type: Schema.Types.ObjectId,
-      ref: 'Tutor',
+      ref: 'User',
       required: true
     },
-    student: {
+    targetUserId: {
```

Comment:
maybe better to name recieverId and recieverRole or we decided to name it target?